AAUnlimited/Functions/Shared/Triggers/Actions.cpp

```diff
@@ -919,6 +942,14 @@ namespace Shared {
 		}
 
 		//int seatPC, int seatPartner
+
+		void Thread::IsConsensualH(std::vector<Value>& params) {
+			int forceVal = params[0].bVal ? 0 : 4;
+			const DWORD offset[]{ 0x376164, 0x44, 0x14, 0x2C, 0x14, 0x9C };
```

Comment:
Where did this come from?

Reply:
Which part? Not quite sure what you are commenting on